css/general.css

```diff
@@ -1,12 +1,98 @@
+
+
 * {
     box-sizing: border-box;
 }
 
+html {
+    height: 100vh;
+}
+
 body {
     margin: 0;
     min-height: 100%;
+    display: flex;
+    flex-direction: column;
+    font-family: Arial, sans-serif;
+    background: url('../image/Cockatoo.webp') no-repeat center center fixed; 
+    background-size: cover;
 }
 
-html {
-    height: 100vh;
+header {
+    text-align: center;
+    padding: 20px;
+    color: white;
+}
+
+header h1 {
+    font-size: 2.5rem;
+}
+
+main {
+    flex: 1;
+    padding: 20px;
+}
+
+.animal-display {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    gap: 50px;
+}
+
+.animal-profile {
+    background: rgba(255, 255, 255, 0.8);
+    border-radius: 10px;
+    padding: 30px;
+    width: 80%; 
+    max-width: 900px;
+    text-align: center;
+    margin: 0 auto;
+    transition: transform 0.3s;
+}
+
+.animal-profile:hover {
+    transform: translateY(-8px);
+}
+
+.animal-profile img {
+    width: 100%;
+    max-width: 100%;
+    height: auto;
+    border-radius: 15px;
+}
+
+.animal-profile h2 {
+    font-size: 2rem;
+    color: #007a4d;
+    margin-bottom: 20px;
+}
+
+footer {
+    background-color: #007a4d;
+    color: white;
+    text-align: center;
+    padding: 10px 0;
+    font-size: 1rem;
+    margin-top: auto;
+}
+
+@media (max-width:1024px)  {
+    
+    body {
+        background: url('../image/Cockatoo.webp') no-repeat center center fixed; 
+    }
+
+    .animal-profile {
+        width: 100%;
+    }
+
+    header h1 {
+        font-size: 2rem;
+    }
+
+    footer {
+        font-size: 0.9rem;
+        padding: 8px 0;
+    }
 }
```

Comment:
The general.css file is supposed to include only the shared styles for every page of the website. That is the body, *, color variables, etc. A new birds.css file has to be created and the code you've written should be moved there, including the changes for the body. Except maybe the new display and flex-direction. Otherwise I don't see nothing wrong with the css, perhaps the color pallete but that is easily changable.